docs/content/Examples-Tutorials-Recipes/Multiple-Data-Sources-With-Same-Schema.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Manage Access to Multiple Data Sources
+permalink: /recipes/multiple-sources-same-schema
+category: Examples & Tutorials
+subCategory: Data sources
+menuOrder: 3
+---
+
+## Use case
+
+We need to access the data from different data sources for different tenants.
+For example, we are the platform for the online website builder, and each client
+can only view their data. The same data schema is used for all clients.
+
+## Configuration
+
+Each client has own database. To manage access to the databases, we need to use
```

Comment:
It's great to explicitly explain which "different data sources" you're going to be connecting to. Something like this:

```suggestion
Each client has its own database. In this recipe, the `X` tenant keeps its data in the remote `ecom` database while the `Y` tenant works with the local database (bootstrapped in the `docker-compose.yml` file) which has the same schema.

To manage access to the databases, we need to use
```

Reply:
I tried to think really hard about what's better:
* just reuse the same credentials and define two connections to identical databases in `driverFactory`
* actually ship a large `ecom.sql` file with this example

Looks like the latter approach is better because we don't "trick" the readers. Let's stick with it!

Reply:
In theory, we can leave only necessary information inside `ecom.sql` (`Users` table in this example). @igorlukanin  what do you think about it?

Reply:
I don't think we should spend time on this.